addons/flashlights/CfgVehicles.hpp

```diff
@@ -0,0 +1,45 @@
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            //todo: add flashlight attach actions
+        }; 
+    };
+    
+    class NATO_Box_Base;
+    class EAST_Box_Base;
+    class IND_Box_Base;
+    class FIA_Box_Base_F;
+
```

Comment:
You don't seem to have Item entries here. I think it would be a good idea to add those. For examples, look at the CfgVehicle file in the medical module and search for Item.